File: Assignmentno7/task_4.c
#include<stdio.h>
void DisplayOdd(int iNo)
{
    int iCnt = 0;
    for(iCnt = 1;iCnt<iNo;iCnt++)
    {
        if((iCnt % 2) != 0)
        {
            printf("%d\t",iCnt);
        }
    }
}

int main()
{
    int iValue = 0;
    printf("enter the number\n");
    scanf("%d",&iValue);

    DisplayOdd(iValue);

    return 0;
}

File: Assignmentno16/task_4.java
import java.util.*;
class Reverse
{
    void ReverseNumber(int iNo)
    {
        int iCnt = 0;
        for(iCnt=iNo;iCnt>=1;iCnt--)
        {
            System.out.println(iCnt);
        }
    }
}

class task_4
{
    public static void main(String A[])
    {
        int Num=0;
        Reverse obj = new Reverse();

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number:");
         Num = sobj.nextInt();
         obj.ReverseNumber(Num);






    
    }
}